src/lib/BeetAPI.js

```diff
@@ -1,10 +1,11 @@
 import * as Actions from './Actions';
 import Queue from './Queue';
-
+import store from '../store/index.js';
 var popupQ = new Queue();
 export default class BeetAPI {
 
     static async handler(request, vueInst) {        
+        console.log(request);
```

Comment:
Add `If DEV` or remove debugging console printouts

Reply:
Will do when committing for merge.

Plan is to add a custom logger throughout for error reports etc.